### PR TITLE
Custom Item Addition: Thomas Calloway OII Coffee Cup

### PR DESCRIPTION
**Pull request by CallowaySCG**
**Link to thread:**  
https://forums.baystation12.net/threads/oii-coffee-cup-thomas-calloway-thomas_calloway.8619/
**Approving maintainer**:  
Eckles
**I have tested this item:**:  
No, I am uncertain how to test the item at this time.

### CODE REVIEW

**Comment by babydoll** [comment]
Note that mugs don't currently work properly in this system; see #207 and https://github.com/Baystation12/Baystation12/blob/dev/code/game/objects/items/weapons/lighter.dm#L193 & https://github.com/Baystation12/custom-items/pull/202

**Comment by CallowaySCG** [comment]
Is there any way to get around that issue, or will it just be a paperweight? It would be neat to put on the desk, even if it does not work. 

Is it just coffee mugs, or is it all liquid containers? I could make a thermos or the like, potentially.

**Comment by babydoll** [comment]
Fixing the issue probably wouldn't be too hard. Head over to #coding channel and ask for help, or I can probably give it a go sometime soon.

File: config/custom_items/items/thomascalloway-oiicoffeecup.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "ckey" :            "thomascalloway",
+  "character_name" :  "Thomas Calloway",
+  "item_name" :       "OII coffee cup",
+  "item_desc" :       "This sleek, dark grey coffee cup is branded with the three-letter initialism of the Office of Interstellar Intelligence: OII.",
+  "item_icon_state" : "oii_coffee_cup",
+  "item_path" :       "/obj/item/weapon/reagent_containers/food/drinks/glass2/coffeecup/OII"
```
Comment (afterthought2) [inline config/custom_items/items/thomascalloway-oiicoffeecup.json]: The correct type path is `/obj/item/weapon/reagent_containers/food/drinks/glass2/coffeecup`.

Comment (CallowaySCG) [reply config/custom_items/items/thomascalloway-oiicoffeecup.json]: You're right! I'm only marginally familiar with GitHub and the process--do I need to delete this pull request and resubmit or can you edit the line?

Comment (CallowaySCG) [reply config/custom_items/items/thomascalloway-oiicoffeecup.json]: I edited the line, this is flagged as outdated. The new path is exactly as you provided it, omitting /OII from the original.

**Comment by babydoll** [comment]
Also the icon that goes in the `custom_items_mob.dmi` is the one that gets displayed in your character's hands. You probably don't want it to be a giant mug.

**Comment by CallowaySCG** [comment]
Reasonable point! I'll edit that and resubmit as well.

**Comment by afterthought2** [comment]
Testing travis; ignore.

**Comment by afterthought2** [comment]
Please try going to the Travis website (https://travis-ci.com/Baystation12/custom-items should work) and clicking log in with github near the top right. After that, try to either force-push or close-reopen this PR.

**Comment by CallowaySCG** [comment]
With Myazaki's help behind the scenes, I squashed my commits and removed the mob icon changes. The mob icon did not need to be included as coffee mugs are designed to be invisible in hands.

**Pull request merged**